sql/codegen_elasticdl.go

```diff
@@ -48,14 +49,48 @@ type elasticDLFiller struct {
 	TrainClause *resolvedTrainClause
 }
 
-func newElasticDLDataConversionFiller(odpsTableName string, featuresList string, batchSize int, numProcesses int) (*elasticDLDataConversionFiller, error) {
+func getFeaturesNames(pr *extendedSelect) ([]string, error) {
```

Comment:
Please be aware that not all selected columns are used as features for training, we can write `select * from some_table train MyModel column a label x ...` and the table may have columns `a,b,c,d...` but only a will be used as training input.

Get feature list from `COLUMN` clause or you can just write some comment here for later update.

Reply:
Yes, this issue will be tracked in the separate TODO issue.